README.md

```diff
@@ -0,0 +1,210 @@
+
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h1><b>BudgetFriend</b></h1>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+## 📗 Table of Contents
+
+- [📖 Budget App ](#-Budget-app-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Test](#test)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 BudgetFriend <a name="about-project"></a>
+
+**BudgetFriend** budget app is a mobile-app and a user-friendly financial management tool that empowers you to take control of your finances with ease. It offers a range of features to help you plan, track, and manage expenses. With this app, you can create  budgets and categorize transactions. 
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Server</summary>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+    <li><a href="https://rubyonrails.org/">Rails</a></li>
+</details>
+<details>
+  <summary>Database</summary>
+    <li><a href="https://www.postgresql.org/">Postgres</a></li>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Create Transaction**
+- **Create account**
+- **Create Payments for transactions**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- <a href="https://budgetapp-isb6.onrender.com/">Live Demo</a>
+## 🚀 Presentation  <a name="Presentation-demo"></a>
+
+- <a href="https://www.loom.com/share/9ff196940ec84edd9ec04131aafce33a">Presentation</a>
+
```

Comment:
Fantastic job with the README documentation!!! 🚀 
All sections are well documented and professional, the getting started section has clear instructions on how to install and run the app locally. 👏🏾 💯 

To improve the documentation and make sure that all the project requirements are met, kindly make sure that there are no broken links. The link the video presentation is broken, please add a working URL.

![image](https://github.com/clin2on3mun/BudgetApp/assets/38088818/61046588-e9d0-4b3e-9be6-9a8582843b0a)